### PR TITLE
Hotfixes and Tweaks for 0.5

### PR DESCRIPTION
**Pull request by helloruhe**
## Changelog:
- 35 new loading screen tips
- New dynasty and character names
- New historical characters
- Added mottos for some dynasties
- Added some new houses and removed some reduntant ones
- New death reasons: Slain by adventurers and merged. Currently used for history only
- 3 new hook types: Minion, Possessed and Mind controlled (in preparation for magic)
- Storm event made less common (this means your characters should die less at sea!!!)
- More names for the Skrog culture
- More nicknames
- Troggs, Drogbars and Grummles have the same lifespan as Dwarves
- The murlocs and naga in the southern Black Morass are now Neptulic
- Jubei'thos and his lands are now Corruptist instead of Sargerite
- Added human ethnicities from vanilla back
- Slightly changed some cultures, like making Gilnean stoic instead of bureaucratic, Gnomish bureaucratic instead of communal etc.
- Added and changed bonuses to some race traits, humans don't have a health bonus but have higher diplomacy, grummles are better caravaneers overall, nathrezim can learn more languages, dwarves can travel the mountains safer, aquatic races can travel the seas safer etc.
- Buffed the Lich King trait
- Fixed High Elven shader issue
- Readded Goblin Hat
- Orc tusks should no longer clip into the lips/mouths of characters
- Fixed indexing issues with the addition of new paradox DLC models
- The centaur impaler MaA is now archer cavalry instead of archer
- Proper icons for the dragon turtle and witch doctor MaA(The dragon turtle's icon was not from its unit icon from Warcraft Reforged, while the witch doctor used the dark troll priest icon)
- Changed the religious requirements for joining the Axis of Awful from the general shaman religion to just the faiths of the original members(Mrligion, Candlist, Fetishist and Salamandrist)
- If you own Legacy of Persia the Carnal Exaltation tenet that Sargerism and Vindicatorism has is replaced with Sacred Murder
- King tier Gnomish republic rulers are called High Tinkers, empire tier Grand Tinkers
- The Exodar title is now the same shade of pink it had in CK2
- A complete Frostwolf family tree
- Assigned court positions for certain characters(Nathanos is Sylvanas' personal champion, Clannia Foxton is Thrall's wetnurse although that's not relevant for the current bookmarks, Brann is Magni's antiquarian etc.)

## 3D Model Additions
Clothes:
- Helm of Domination
- Alliance helmet (7 variations)
- Alliance armor (7 variations)
- Bandana (4 Variations)
- Whitemane hat (Female Only)
- Dwarf crowns (2 variations)
- Shadowforge crown
- Ironforge crown
- Dwarf war helmet
- Whitemane robe (Female Only)

Artifacts:
- Doomhammer (2 variations)
- Felo'melorn
- Twilight Hammer
- GoreHowl

## Developer changelog:
- this is for small tweaks and changes, feel free to commit, but add the changes in the changelog so we know what to test.

## Tests:
- [ ] There are no errors in `wc` files in `Documents\Paradox Interactive\Crusader Kings III\logs\error.log` except `portrait_decals.cpp:101`
- [ ] The mod takes less than 5.5 GB in the Task Manager (Windows)

# How to test:

### CODE REVIEW

File: localization/english/load_tips/load_tips_l_english.yml [hunk 1]
```diff
@@ -97,9 +97,34 @@
...
+ LOADING_TIP_102:0 "“Glad you could make it, Uther.”\n #bold —Arthas Menethil, to Uther the Lightbringer#!"
+ LOADING_TIP_103:0 "“Watch your tone with me, boy. You may be the prince, but I'm still your superior as a paladin!”\n #bold —Uther the Lightbringer, to Arthas Menethil#!"
+ LOADING_TIP_104:0 "“As if I could forget. Listen, Uther, there's something about the plague you should know...”\n #bold —Arthas Menethil, to Uther the Lightbringer#!"
+ LOADING_TIP_105:0 "“Oh, no. We're too late. These people have all been infected! They may look fine now, but it's just a matter of time before they turn into the undead!”\n #bold —Arthas Menethil, to Uther the Lightbringer#!"
+ LOADING_TIP_106:0 "“What?”\n #bold —Uther the Lightbringer, to Arthas Menethil#!"
```
Comment (edmund-h) [inline localization/english/load_tips/load_tips_l_english.yml:108]: you literally put the whole dialogue in LMAOOOOO

File: localization/english/load_tips/load_tips_l_english.yml [hunk 2]
```diff
@@ -97,9 +97,34 @@
...
+ LOADING_TIP_122:0 "“In the Age of Chaos two factions battled for dominance.”\n #bold —Warcraft: Orcs & Humans cinematic intro#!"
+ LOADING_TIP_123:0 "“We never paid any heed to the ancient prophecies... Like fools we clung to the old hatreds, and fought as we had for generations. Until the day the sky rained fire... and a new enemy came among us. We stand now, upon the brink of destruction... for the Reign of Chaos has come at last.”"
+ LOADING_TIP_124:0 "“Keep the body strong, and the mind will follow.”\n #bold —Mokimo the Strong#!"
+ LOADING_TIP_125:0 "“The champions who fought at our side assured the survival of our world... but now, we must see it, with mortal eyes. We dragon aspects have fulfilled our great purpose and our ancient power... is expended. But though our day draws to an end, life endures... and new generations will be born. Today's victory belongs to all who stood against the shadow. You are Azeroth's true Guardians... And the future of this world is in your hands... For the dawning of the age of mortals has begun.”\n #bold —Alexstrasza the Life-Binder#!"
```
Comment (edmund-h) [inline localization/english/load_tips/load_tips_l_english.yml:127]:
```suggestion
 LOADING_TIP_125:0 "“The champions who fought at our side assured the survival of our world... but now, we must see it, with mortal eyes. We dragon aspects have fulfilled our great purpose and our ancient power... is expended. But though our day draws to an end, life endures... and new generations will be born. Today's victory belongs to all who stood against the shadow. You are Azeroth's true Guardians... And the future of this world is in your hands... For the dawning of the age of mortals has begun.”\n #bold —Alexstrasza the Life-Binder#!"
 LOADING_TIP_126:0 "“I've got a flying machine!”\n #bold — Anonymous Gnome, Battle of Tol Barad, Second War#!"
 LOADING_TIP_127:0 "“'Join the army,' they said. 'See the world,' they said. I'd rather be sailing!”\n #bold — Anonymous Footman, Battle of Hillsbrad, Second War#!"
 LOADING_TIP_128:0 ""We're ready, master—""#italic I'm#! not ready!"#bold — Anonymous Ogre, Siege of Stromgarde, Second War#!"
 LOADING_TIP_129:0 ""You think Lothar's death was my fault, don't you?"#bold — Turalyon, Siege of Nethergarde Keep, Allied Invasion of Draenor#!"
 LOADING_TIP_130:0 ""These abominations are responsible for the deaths of dozens of squirrels! They deserve no mercy!"#bold — King Mrgl-Mrgl, Night Elf Anthropologist, Second Siege of Mount Hyjal#!"
 
```

**Review by edmund-h — APPROVED** [review]

File: localization/english/load_tips/load_tips_l_english.yml (hunk 1 above)
Comment (Grekopithikos) [reply localization/english/load_tips/load_tips_l_english.yml:108]: I got the urge to make an easter egg shitpost but if it's too wacky I can remove it

Comment (helloruhe) [reply localization/english/load_tips/load_tips_l_english.yml:108]: nah this is funny as hell keep it

File: history/characters/16000_gnome.txt [hunk 3]
```diff
@@ -1990,6 +1990,162 @@
 	719.12.7={ death=yes }
 }
 
+# dynasty = 10054 [Springcrank]
+16900={
+	name=Gearspinner
+	dynasty=10054
+	culture=gnome religion=rationalism
```
Comment (Grekopithikos) [inline history/characters/16000_gnome.txt:1997]: Are these gnomes canon?

Comment (edmund-h) [reply history/characters/16000_gnome.txt:1997]: no, i just spent like 3 hours making them for the mechagon thing and was like, might as well put them in the mod

**Comment by helloruhe** [comment]
@Grekopithikos add ur changes to desc please

**Review by ElMariuso — CHANGES_REQUESTED** [review]

File: common/coat_of_arms/coat_of_arms/wc_religious_titles.txt [hunk 4]
```diff
@@ -78,4 +78,56 @@ k_forsaken_cult = {
...
+d_red_wyrmcult = {
+	pattern = "pattern_solid.dds"
+	color1 = "red"
+	colored_emblem = {
+		texture = "ce_wyvern.dds"
+		color1 = "red"
+		color2 = "red"
+		color3 = "red"
+	}
+}
```
[the remark this answers is not in the document]
Comment (Grekopithikos) [reply common/coat_of_arms/coat_of_arms/wc_religious_titles.txt:113]: what other titles

[the remark this answers is not in the document]
Comment (Grekopithikos) [reply common/coat_of_arms/coat_of_arms/wc_religious_titles.txt:113]: idk these ones were supposed to be placeholders in the first place

[the remark this answers is not in the document]
Comment (ElMariuso) [reply common/coat_of_arms/coat_of_arms/wc_religious_titles.txt:113]: well, ok

**Comment by skesus** [comment]
Looks good to go except Whitemane's clothing has her with no booty and her crotch showing.

**Review by skesus — APPROVED** [review]
only thing i noticed was whitemane's outfit has crotch showing and no booty.everything else checks out, no new errors spotted.

**Pull request merged**